game/src/Game/GameStart.tscn

```diff
@@ -57,7 +57,7 @@ process_mode = 2
 disable_3d = true
 title = "VIC2_DIR_DIALOG_TITLE"
 size = Vector2i(935, 159)
-ok_button_text = "VIC2_DIR_DIALOG_SELECT"
+ok_button_text = "Select Current Folder"
```

Comment:
I did not touch this. Why has this changed?

Reply:
This gets changed upon localisation by the Godot editor due to a bug, there's a comment explaining it in a GDScript file somewhere. Unfortunately we just have to ignore this and not include it in PRs, hopefully it'll be fixed with Godot 4.3.